### PR TITLE
Publish updates from master

### PR DESCRIPTION
**Pull request by usha-mandya**
Daily PR to sync updates from master

### CODE REVIEW

File: release-notes/docker-compose.md [hunk 1]
```diff
@@ -5,6 +5,93 @@ keywords: release notes, compose
...
+
+- Lowered the severity to `warning` for instances in which `down` attempts to remove a non-existent image.
+
+- Mandated the use of improved API fields for project events, when possible.
+
+- Updated `setup.py` for modern `pypi/setuptools`, and removed `pandoc` dependencies.
+
+- Removal of `Dockerfile.armhf`, which is no longer required.
+
+### Bugfixes
```
Comment (traci-morrison) [inline release-notes/docker-compose.md]: s/Bugfixes/Bug fixes

File: release-notes/docker-compose.md [hunk 2]
```diff
@@ -5,6 +5,93 @@ keywords: release notes, compose
...
+
+- Made container service color deterministic, including the removal of the color red.
+
+- Fixed non-ASCII character errors (Python2 only).
```
Comment (traci-morrison) [inline release-notes/docker-compose.md]: s/Python2/Python 2

File: release-notes/docker-compose.md [hunk 3]
```diff
@@ -5,6 +5,93 @@ keywords: release notes, compose
...
+
+- Changed image sizing to decimal format, to align with Docker CLI.
+
+- `tty` size acquired via Python Posix support.
```
Comment (traci-morrison) [inline release-notes/docker-compose.md]: s/Posix/POSIX

File: release-notes/docker-compose.md [hunk 4]
```diff
@@ -5,6 +5,93 @@ keywords: release notes, compose
...
+
+- Fixed the failure check in the `parallel_execute_watch function`.
+
+- Fixed the race condition that occurs following the pulling of an image.
+
+- Fixed error on duplicate mount points (a configuration error message now displays).
+
+- Fixed the merge on `networks` section.
+
+- Compose container is always connected to `stdin`.
```
Comment (traci-morrison) [inline release-notes/docker-compose.md]: Should there be more information here? Is this a fix or a known issue?

**Comment by KoryKessel-Docker** [comment]
Sent Usha a few small edits via email about an hour ago...including those comments here now (inspired by Traci):

**Features**
Added the following tags: `docker-compose:latest`, `docker-compose:<version>-alpine`, and `docker-compose:<version>-debian`.
Added a `--quiet` option  and a `--no-rm` option to the `docker-compose build` command. (In making this edit — a merging of two bullets into one — be careful to also delete "Added a `--no-rm` option to the `docker-compose build' command.")

**Bug Fixes**
Fixed same file `extends` optimization.
Fixed `stdin_open`.
Fixed bash completion for the `docker-compose build --memory` command.
Fixed the misleading environmental variables warning that occurs when the `docker-compose exec` command is performed.

**Comment by usha-mandya** [comment]
PR with updates to master - https://github.com/docker/docker.github.io/pull/9966

File: release-notes/docker-compose.md (hunk 1 above)
Comment (usha-mandya) [reply release-notes/docker-compose.md]: Fixed

File: release-notes/docker-compose.md (hunk 2 above)
Comment (usha-mandya) [reply release-notes/docker-compose.md]: Fixed.

File: release-notes/docker-compose.md (hunk 3 above)
Comment (usha-mandya) [reply release-notes/docker-compose.md]: Fixed.

File: release-notes/docker-compose.md (hunk 4 above)
Comment (usha-mandya) [reply release-notes/docker-compose.md]: Updated the note.

**Pull request merged**